src/components/LayoutComponent/page_not_found.module.css

```diff
@@ -0,0 +1,37 @@
+.pages_not_found {
```

Comment:
Use a consistent format for naming CSS classes: either camelCase, kebab-case, or snake_case. Do not mix formats.